R/pbdb_querys.R

```diff
@@ -761,32 +772,33 @@ pbdb_ref_collections <-function(...){
 #' instead of Taxon records. One record is returned per reference, 
 #' even if it is associated with multiple taxa.
 #' 
-#' @usage pbdb_ref_taxa (...)
+#' @usage pbdb_ref_taxa(...)
 #' @param ... arguments passed to the API. See all available arguments in
-#'   \url{http://paleobiodb.org/data1.1/taxa/refs}
+#'   \url{https://paleobiodb.org/data1.2/taxa/refs}
 #'   \itemize{
 #'     \item \emph{name}: returns information about the most fundamental 
 #'       taxonomic name matching this string. 
 #'       The \% and _ characters may be used as wildcards.
 #'     \item \emph{id}: returns information about the taxonomic name 
 #'       corresponding to this identifier. You may not specify both 
 #'       name and id in the same query.
-#'     \item \emph{exact}: if this parameter is specified, then the taxon exactly 
-#'       matching the specified name or identifier is selected, 
-#'       rather than the senior synonym which is the default.
 #'     \item \emph{show}: show extra variables
-#'     \item \emph{rel}: set rel="synonyms" to select all synonyms of 
-#'       the base taxon or taxa; rel="children" to select the 
-#'       taxa immediately contained within the base taxon or taxa; 
-#'       rel="common_ancestor" to select the most specific taxon 
-#'       that contains all of the base taxa.
+#'     \item \emph{rel}: set rel = "synonyms" to select all synonyms
+#'       of the base taxon or taxa; rel = "children" to select the
+#'       taxa immediately contained within the base taxon or taxa; rel
+#'       = "all_children" to select all taxa contained within each
+#'       matching taxon and within all synonymous taxa; rel =
+#'       "all_parents" to select all taxa that contain any of the
+#'       matching taxa.
 #'     \item \emph{extant}: TRUE/FALSE to select extant/extinct taxa.
 #'   }
 #' @return a dataframe with references from a list of taxa
 #'  
 #' @export 
 #' @examples \dontrun{
-#'   pbdb_ref_taxa (name="Canidae", vocab="pbdb", show=c("attr", "app", "size", "nav")) 
+#'   pbdb_ref_taxa(
+#'     name = "Canidae", vocab = "pbdb", show = c("attr", "app", "size", "nav")
```

Comment:
Accroding to [PBDB Data Service 1.2 v2](https://paleobiodb.org/data1.2/) > [Taxonomic names](https://paleobiodb.org/data1.2/taxa_doc.html) > [Bibliographic references for taxa](https://paleobiodb.org/data1.2/taxa/refs_doc.html), available values for **parameter _show_** are follow**ing: counts, formatted, both, comments, ent, entname, crmod**.

Reply:
Thank you for pointing this out. It seems that the JSON object returned by the API can have a `"warnings"` property that the `paleobioDB` package simply ignores, so many of these examples, while they are not well formed, run without errors and return a dataframe (so I did not realize they were wrong). For now, I will simply correct the examples in the documentation, but the package should probably report these warnings.